### PR TITLE
22 create home page

### PR DESCRIPTION
**Pull request by TheDoplarEffect**
A Hopefully complete Home page now that all the elements are included.

I added a bonus Search Element

This si a headless UI component that you can search for any film in the API.  There is also a little emoji of the genre of the movie displayed next to each option.

Unfortunately this meant I had to re-write large sections of the Emoji component that was already done for it to take numbers (what the API gives instead of a string.)
The functionality is still the same there is just an Object instead of an ENUM.

Also had to add text-3xl.  This is the only time we will use it, if we use the measurements on the Figma file the icon buttons just look plain ridiculous.

### CODE REVIEW

File: src/components/API.tsx [hunk 1]
```diff
@@ -13,4 +13,26 @@ export const API = {
...
   },
+  searchResult: async (input: string) => {
+    const info: Movie[] = await axios
+      .get(
+        `https://api.themoviedb.org/3/search/movie?api_key=5de0d3a9c085fde70b8c91f6f6a927f3&include_adult=false&query=${input}`
+      )
+      .then((res) => {
+        return res.data.results;
+      });
+    const returnStuff = info.map((x) => {
```
Comment (ReallyBoard123) [inline src/components/API.tsx:24]: returnStuff makes it difficult to understand what is "stuff" here

**Comment by TheDoplarEffect** [comment]
Changed the name of the variable to returnObject

File: src/routes/Home.tsx [hunk 2]
```diff
@@ -1,7 +1,69 @@
...
+      <div className="w-full text-white text-2xl flex justify-center items-center">
+        <SearchBar></SearchBar>
+      </div>
+
+      <div className="w-full h-full flex justify-around items-center flex-col gap-4">
+        <div className="flex flex-row w-full px-5 justify-between">
+          <div className="text-title text-white-dimmed">Genre</div>
+          <div className="">
+            <a
+              className="text-description text-yellow flex flex-row items-center gap-4"
```
Comment (ReallyBoard123) [inline src/routes/Home.tsx:30]: color for "See All" does not matches the one from the figma file. either add opacity, or create a yellow-dimmed

**Review by ReallyBoard123 — CHANGES_REQUESTED** [review]
the navigation bar appears at the bottom of the home page, but somehow doesn´t have the bg props. please see that the navigation bar is also visible. everything else works great. I liked the combobox

**Comment by TheDoplarEffect** [comment]
Reviewed and fixed on iPhoneSE

**Review by ReallyBoard123 — APPROVED** [review]

**Pull request merged**